README.md

```diff
@@ -52,9 +52,12 @@ e.g. `sbt "resourceValidator/run --dryRun --all"`
 
 Currently, `com.broadinstitute.dsp.zombieMonitor.DbReaderSpec` and `com.broadinstitute.dsp.resourceValidator.DbReaderSpec` are not run in CI, hence make sure you run them manually before merging any PRs.
 
-Once a PR is merged, there will be a PR created in [terra-helm](https://github.com/broadinstitute/terra-helm). 
-Get this PR merged, and another automatic commit will bump leonardo's chart version. This will trigger another automatic commit 
-in [terra-helmfile](https://github.com/broadinstitute/terra-helmfile), note this commit will only auto bump `dev` and `perf`, and will be auto-merged. Once the terra-helmfile PR is auto-merged, go to [argo](https://ap-argocd.dsp-devops.broadinstitute.org/applications) (you need to be on VPN to access argo), and click `SYNC APPS` button on the left upper corner for dev and perf leonardo deploys (select `PRUNE` option as well).
+Once a PR is merged, a PR will be automatically created in [terra-helm](https://github.com/broadinstitute/terra-helm). 
+
+Get this PR merged (you can merge it yourself) and another automatic commit will bump leonardo's chart version. This will trigger another automatic commit 
+in [terra-helmfile](https://github.com/broadinstitute/terra-helmfile), note this commit will only auto bump `dev` and `perf`, and will be auto-merged.
```

Comment:
Nit:
```suggestion
in [terra-helmfile](https://github.com/broadinstitute/terra-helmfile). Note that this commit will only auto-bump `dev` and `perf`, and will be auto-merged.
```

Reply:
oh my you were fast to review! haha I am hoping to make a few more edits, will update you once i commit those!

Reply:
my bad that I jumped on it despite the fact that you had marked it as `Draft` 🤦🏼

Reply:
no problem! appreciate the feedback!